This patch is a backport from lineak-cvs that avoids a kde-buildtime-dep

--- lineak_xosdplugin-0.8.2/configure.in.in	2004-11-02 14:56:20.000000000 +0100
+++ lineak_xosdplugin/configure.in.in	2004-11-17 04:32:14.000000000 +0100
@@ -1,7 +1,38 @@
-#MIN_CONFIG(3)
+AC_INIT(acinclude.m4) dnl a source file from your sub dir
+
+dnl This is so we can use kde-common
+AC_CONFIG_AUX_DIR(admin)
+
+dnl This ksh/zsh feature conflicts with `cd blah ; pwd`
+unset CDPATH
+
+dnl Checking host/target/build systems, for make, install etc.
+AC_CANONICAL_SYSTEM
+dnl Perform program name transformation
+AC_ARG_PROGRAM
 
 AM_INIT_AUTOMAKE(lineak_xosdplugin,0.8.2)
 
+AC_PREFIX_DEFAULT(/usr)
+
+if test "x$prefix" = "xNONE"; then
+  prefix=$ac_default_prefix
+  ac_configure_args="$ac_configure_args --prefix $prefix"
+fi
+
+AM_CONFIG_HEADER(config.h)
+AC_CHECK_COMPILERS
+AC_ENABLE_SHARED(yes)
+AC_ENABLE_STATIC(no)
+
+KDE_PROG_LIBTOOL
+all_libraries="$all_libraries $USER_LDFLAGS"
+all_includes="$all_includes $USER_INCLUDES"
+AC_SUBST(all_includes)
+AC_SUBST(all_libraries)
+
+AC_SUBST(AUTODIRS)
+
 AC_LINEAK_CHECK_INCLUDES
 AC_LINEAK_CHECK_LIBS
 AC_LINEAK_FIND_PLUGINDIR
